src/app/code/community/Omise/Gateway/Model/Payment/Offsiteconveniencestore.php

```diff
@@ -0,0 +1,93 @@
+<?php
+class Omise_Gateway_Model_Payment_Offsiteconveniencestore extends Omise_Gateway_Model_Payment_Offline_Payment
```

Comment:
Should it be called `Omise_Gateway_Model_Payment_Offlineconveniencestore`?

Reply:
✋ I think it would be nice to call `Omise_Gateway_Model_Payment_Offlinekonbini` instead. 
As this payment method is specifically made for Japanese convenience store. I think to use its common name would be less confuse (i.e. 3rd-party Japanese developers can find the related code and files easier)